packages/gatsby/cache-dir/ensure-resources.js

```diff
@@ -16,6 +17,7 @@ class EnsureResources extends React.Component {
     this.state = {
       location: { ...location },
```

Comment:
Where did we ever end up on this? Is this needed?

Reply:
hah, probably not! just don't use `git blame` - it tells lies!